README.md

```diff
@@ -27,7 +28,40 @@ $ pip install quanteda
 
 ## Usage
 
-- TODO
+Follow the instructions below to run the unit tests of the package
+
+### Setup
+
+1. Clone this repository.
+
+```bash
+$ git clone git@github.com:UBC-MDS/quanteda.git
+$ cd quanteda/
+``` 
+
+2. Install the virtual environment.
+
+```bash
+$ conda env create -f environment.yml
+```
+
+3. Activate the installed environment:
+
+```bash
+$ conda activate quanteda
+```    
+
+4. Install the package.
+
+```bash
+$ poetry install
+```
```

Comment:
Got an error when I tried this step: 
> Warning: poetry.lock is not consistent with pyproject.toml. You may be getting improper dependencies. Run `poetry lock [--no-update]` to fix it.
Not sure if someone else had this issue?

Reply:
same!

Reply:
I checked the poetry.lock on main which was working for me. Looks like it only contains auto generated content of metadata, while the one in this PR contains some packages/dependence etc.. @JohnShiuMK I wonder if these packages were added manually?

Reply:
<img width="754" alt="image" src="https://github.com/UBC-MDS/quanteda/assets/100008826/0b3a07c5-af61-4cd6-859f-a6f5b10bd59e">
vs 
<img width="939" alt="image" src="https://github.com/UBC-MDS/quanteda/assets/100008826/f38e44f7-d803-4799-8d2b-5b367c3c452b">

Reply:
If I replace the lock file with the one (auto generated) in main brach, I have no warning message

Reply:
@dorisyycai @meretelutz 
Oh, I commented that in another PR:

> Btw, I am not sure if you guys have this problem. I need to run poetry install to reflect any changes from the pyproject.toml, but that would lead to error unless I delete the poetry.lock file

I'm thinking whether we should avoid committing the poetry.lock. If we don't have the poetry.lock file there, and run the `poetry install`, I got no warnings / errors

Reply:
Got it, seems like thats a good way to avoid these issues